index.html

```diff
@@ -14,6 +14,43 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="resume-main">
+  <header>
+    <div id="navigation">
+      <a href="home">Home</a>
+      <a href="about">About</a>
+      <a href="Portfolio">Portfolio</a>
+      <a href="contact">Contact</a>
+    </div>
+  </header>
+  
+  <div class="Introduction">
+      <img src="http://www.runsilly.com/wp-content/uploads/2014/02/minnie-mouse.gif"
+      alt="minnie mouse in red polka dot dress">
+      <p>Fashion School Graduate from Fashion Institute of Design & Merchandising and Founder of the Bow</p>
+  </div>
+<div class="middle">
+  <div class="Skills">
+    <h1>Skills</h1>
+      <li>Bow-making</li>
+      <li>Management</li>
+      <li>Problem-Solving</li>
+    </div>
+    <div class="Work-History">
+        <h1>Work History</h1> 
+            <li>Minnie's Bow-Tique</li>
```

Comment:
Great working structuring your html. I would recommend installing an extension like `Beautify` or `Prettier` to clean up some of this spacing issue.